impedancefitter/fitter.py

```diff
@@ -153,16 +154,19 @@ def __init__(self, inputformat, directory=None, **kwargs):
         if self.fileList is None:
             self.fileList = sorted(os.listdir(self.directory))
         read_data_sets = 0
+
         for filename in self.fileList:
```

Comment:
Here is kind of a broken logic now: with the dataframe, there is no need to go through a file list.
I would separate this.
If you do not have an idea, let me know and I will do it.